einhorn.gemspec

```diff
@@ -23,5 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'chalk-rake'
   gem.add_development_dependency 'subprocess'
 
+  gem.add_runtime_dependency 'ffi'
```

Comment:
I should mention that I don't know anything about either and certainly haven't used them, so it's your call